lib/octokit/client.rb

```diff
@@ -338,7 +338,8 @@ def request(method, path, data, options = {})
         end
       end
 
-      @last_response = response = agent.call(method, URI::Parser.new.escape(path.to_s), data, options)
+      response = agent.call(method, URI::Parser.new.escape(path.to_s), data, options)
+      @last_response = response if !path.include?('rate_limit')
```

Comment:
This is probably a bad way to detect this. Maybe we should do `!path.start_with('/rate_limit')`?